sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 12.0.0-beta.13 (Unreleased)
+## 12.0.1 (2020-03-11)
```

Comment:
still yet to update a few things depending on small changes that may need to go in

Reply:
Datalake is 12.0.1 to allow it to show up higher than the preview releases that were done for 12.0.0

Reply:
@JimSuplizio just wanted to make sure this was the right thing to do

Reply:
@gapra-msft yes, being that datalake had released "-preview" libraries at 12.0.0, this was the right call.